src/chart/Scatter.js

```diff
@@ -0,0 +1,171 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "./XYAxis", "../api/INDChart", "css!./Scatter"], factory);
+    } else {
+        root.chart_Column = factory(root.d3, root.chart_XYAxis, root.api_INDChart);
+    }
+}(this, function (d3, XYAxis, INDChart) {
+    function Scatter(target) {
+        XYAxis.call(this);
+        INDChart.call(this);
+    }
+
+    Scatter.prototype = Object.create(XYAxis.prototype);
+    Scatter.prototype._class += " chart_Scatter";
+    Scatter.prototype.implements(INDChart.prototype);
+
+    Scatter.prototype.publish("paletteID", "default", "set", "Palette ID", Scatter.prototype._palette.switch(),{tags:['Basic','Shared']});
+    Scatter.prototype.publish("dotRepresentation", "cross", "set", "Selects the representation for the data", ["circle", "rectangle", "cross"]);
+    Scatter.prototype.publish("pointSize", 8, "number", "pointSize");
+
+    Scatter.prototype.updateChart = function (domNode, element, margin, width, height) {
+        var context = this;
+
+        this._palette = this._palette.switch(this.paletteID());
+
+        var column = this.svgData.selectAll(".data")
+            .data(this._data, function (d) { var k = d.filter(function(d, i) {return i > 0;}); return k + Math.random(); })
+        ;
+
+        column.enter().append("g")
+            .attr("class", "data")
+        ;
+
+        column
+            .each(function (d, i) {
+                var element = d3.select(this);
+                var point = element.selectAll(".point").data(d.filter(function(d, i) {return i > 0;}));
+                var point1 = element.selectAll(".point").data(d.filter(function(d, i) {return i > 0;}));
+                switch (context.dotRepresentation()) {
+                    case "circle" :
+                        var title = point
+                          .enter().append("circle")
+                            .attr("class", "point")
+                            .attr("r", context.pointSize()/2)
```

Comment:
This should be set in the "update" cycle not the "enter()" cycle (and it only appears to be working now due to the random above)